### PR TITLE
Homework №9

### PR DESCRIPTION
**Pull request by IrinaaZakutilina**

### CODE REVIEW

File: src/main/java/org/example/homework9/Contact.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    public String getPhoneNumber() {
+        return phoneNumber;
+    }
+
+    public static Contact createContact(String name, String phoneNumber) {
+        return new Contact(name, phoneNumber);
+    }
+}
+
```
Comment (alexshamrai) [inline src/main/java/org/example/homework9/Contact.java]: remove empty lines in the end, please

File: src/main/java/org/example/homework9/MobilePhone.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
+package org.example.homework9;
+import java.util.ArrayList;
+
+public class MobilePhone {
+    private final ArrayList<Contact> myContacts;
```
Comment (alexshamrai) [inline src/main/java/org/example/homework9/MobilePhone.java]: one field is absent in the class, please read the task description carefully

Comment (alexshamrai) [inline src/main/java/org/example/homework9/MobilePhone.java]: it should not be final, `myContacts` is modifiable
https://www.baeldung.com/java-final#3-final-fields

File: src/main/java/org/example/homework9/MobilePhone.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+    public MobilePhone(String myNumber) {
+        this.myContacts = new ArrayList<>();
+    }
```
Comment (alexshamrai) [inline src/main/java/org/example/homework9/MobilePhone.java:11]: empty line is needed

**Review by alexshamrai — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/homework9/MobilePhone.java (hunk 3 above)
Comment (IrinaaZakutilina) [reply src/main/java/org/example/homework9/MobilePhone.java:11]: I added changes to the code based on your comments

**Pull request merged**